src/components/Modal/Modal.jsx

```diff
@@ -1,25 +1,51 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Base from '../Base.jsx';
 import style from './style.scss';
+import Header from '../Header/Header';
+import getDataAttrs from '../../getDataAttrs';
 
 class Modal extends React.Component {
   render() {
     let comp = null;
+    const {
+      title,
+      actions,
+      data,
+      children,
+      onDismiss,
+      minWidth,
+    } = this.props;
     if (this.props.visible) {
```

Comment:
As a style thing, I usually prefer to limit the use of early returns because it makes the code easier to follow (in this very simple case, not a big deal obvi). If there's a strong preference for that style, we can change.